src/components/Spinner/Spinner.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { TPAComponentsWrapper } from '../../test/utils';
+import { spinnerDriverFactory } from './Spinner.driver';
+import { Spinner } from './';
+import { spinnerTestkitFactory } from '../../testkit';
+import { spinnerTestkitFactory as enzymeSpinnerTestkitFactory } from '../../testkit/enzyme';
+
+describe('Spinner', () => {
+  const createDriver = createUniDriverFactory(spinnerDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<Spinner />);
+    expect(await driver.exists()).toBe(true);
+  });
+
+  it('should use mobile design', async () => {
+    const driver = createDriver(
+      TPAComponentsWrapper({ mobile: true })(<Spinner />),
+    );
+    expect(await driver.isMobile()).toBe(true);
+  });
```

Comment:
Again, I think this can be removed, since there's no special design for mobile.
In fact, all references to mobile can be removed for this component I guess